addons/aws/1.0.1/kubeadm-cluster-config-v1beta2.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+metadata:
+  name: kubeadm-cluster-configuration
+clusterName: $KUBERNETES_CLUSTER_NAME
+networking:
+  podSubnet: $POD_CIDR
+  serviceSubnet: $SERVICE_CIDR
```

Comment:
this is already set [here](https://github.com/replicatedhq/kURL/blob/e7dc0cda408ed79da1d7f7411f02ddf4ca1368a0/scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml#L21). if its not necessary remove